packages/gafl-webapp-service/src/pages/layout/layout.njk

```diff
@@ -24,6 +24,11 @@
 <meta http-equiv="Cache-Control" content="no-store" />
 <meta http-equiv="Pragma" content="no-cache">
 <meta http-equiv="Expires" content="-1">
+{% if gtmContainerId %}
+    <!-- Google Tag Manager -->
+    <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start': new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','{{gtmContainerId}}');</script>
+    <!-- End Google Tag Manager -->
+{% endif %}
```

Comment:
Might be over-egging this, but I wonder if it would be worth putting something in the logs if the gtmContainerId isn't present? We could put something in the page console.info, as well as in the GAFL process logs

Reply:
Added logging during server startup